CmswebApI/Controllers/CoursesController.cs

```diff
@@ -119,5 +117,30 @@ public async Task<ActionResult<CourseDto>> UpdateCourseAsync(int courseID, Cours
                 return StatusCode(StatusCodes.Status500InternalServerError, ex.Message);
             }
         }
+
+        [HttpDelete("{courseID}")]
+        public async Task<ActionResult<CourseDto>> DeleteCourseByIdAsync(int courseID)
+        {
+            try
+            {
+                if (!await _cmsrepository.IsCourseExistsAsync(courseID))
+                {
+                    return NotFound();
+                }
+                Course course = await _cmsrepository.DeleteCourseByIdAsync(courseID);
+
+                if (course == null)
+                {
+                    return BadRequest();
```

Comment:
Just use the return result by ICMSRepository and return ok() when deletion was successful